docs/getting-started/quick-start.md

```diff
@@ -31,9 +31,9 @@ which kontena > /dev/null && . "$( kontena whoami --bash-completion-path )"
 
 In order to use Kontena, you'll need a Kontena Master. If you don't have an existing Kontena infrastructure in place, you need to install your own. A Kontena Master can be provisioned to any cloud platform. It' s also possible to run a Kontena Master on your own local development environment for testing purposes.
 
-The easiest (and preferred) way to provision Kontena Master is to use the built-in Kontena Master provision feature of Kontena CLI. In this guide, we will provision Kontena Master to local development environment using [Vagrant](https://www.vagrantup.com/). If you want to install Kontena Master to some other environment, please see [Installing Kontena](installing/) documentation.
+The easiest (and preferred) way to provision Kontena Master is to use the built-in Kontena Master provision feature of Kontena CLI. In this guide, we will provision Kontena Master to the local development environment using [Vagrant](https://www.vagrantup.com/). If you want to install Kontena Master to some other environment, please see [Installing Kontena](installing/) documentation.
```

Comment:
Just wondering, should it perhaps be `a local development environment` or `your local development environment` or maybe better yet: `we  will provision a Kontena Master to your local development environment`